recipe/0068-newspaper/newspaper_title-collection.json

```diff
@@ -0,0 +1,96 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "type": "Collection",
+  "id": "{{ id.url }}",
+  "within": "http://data.theeuropeanlibrary.org/Collection/a0620",
```

Comment:
Newspaper group review: we think this should be `partOf` rather than `within`. partOf "MUST have the id and type properties, and SHOULD have the label property." We also wondered if it would be better to remove as it doesn't seem to resolve to a IIIF Collection.

Reply:
The implementation of IIIF Collections is on hold at Europeana, so I believe it is better to remove it.

Reply:
removed